robots/commenter/main.go

```diff
@@ -115,29 +122,33 @@ func parseHTMLURL(url string) (string, string, int, error) {
 	return mat[1], mat[2], n, nil
 }
 
-func makeQuery(query string, includeArchived, includeClosed bool, minUpdated time.Duration) (string, error) {
-	parts := []string{query}
-	if !includeArchived {
-		if strings.Contains(query, "archived:true") {
-			return "", errors.New("archived:true requires --include-archived")
+func makeQuery(queries string, includeArchived, includeClosed bool, minUpdated time.Duration) ([]string, error) {
```

Comment:
I know I suggested this in part, maybe it would be cleaner / give a smaller diff if we just call this func in a for loop and build the array one layer above

Reply:
But I think the dedup of the already found issues will re-create the diff then